velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -1511,14 +1511,13 @@ INSTANTIATE_TEST_SUITE_P(
         common::CompressionKind::CompressionKind_LZ4,
         common::CompressionKind::CompressionKind_GZIP));
 
-TEST_F(PrestoSerializerTest, deserializeSingleColumn) {
-  // Verify that deserializeSingleColumn API can handle all supported types.
+TEST_F(PrestoSerializerTest, serdeSingleColumn) {
   static const size_t kPrestoPageHeaderBytes = 21;
   static const size_t kNumOfColumnsSerializedBytes = sizeof(int32_t);
   static const size_t kBytesToTrim =
       kPrestoPageHeaderBytes + kNumOfColumnsSerializedBytes;
 
-  auto testRoundTripSingleColumn = [&](const VectorPtr& vector) {
+  auto testDeserializeSingleColumnRoundTrip = [&](const VectorPtr& vector) {
```

Comment:
Maybe call the lambdas testSerializeRoundTrip and testSerializeSingleColumnRoundTrip to make the difference between the 2 lambdas more clear. Also write a comment to indicate that between serialize and serializeSIngleColumn is the PrestoPage header and number of columns section from the serialized data.